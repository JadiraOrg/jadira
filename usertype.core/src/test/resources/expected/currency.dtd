<!ELEMENT dataset (
    CURRENCY*)>

<!ELEMENT CURRENCY EMPTY>
<!ATTLIST CURRENCY
    ID CDATA #REQUIRED
    CURRENCY CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

