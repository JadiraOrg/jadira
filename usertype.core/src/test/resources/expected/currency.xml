<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "currency.dtd">
<dataset>
  <currency ID="0" CURRENCY="EUR" NAME="test_0"/>
  <currency ID="1" CURRENCY="USD" NAME="test_1"/>
  <currency ID="2" CURRENCY="GBP" NAME="test_2"/>
  <currency ID="3" CURRENCY="SAR" NAME="test_3"/>
  <currency ID="4" NAME="test_4"/>
</dataset>